Assets/MixedRealityToolkit.Services/InputSystem/INearInteractionTouchable.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Input
+{
+    public interface INearInteractionTouchable
```

Comment:
can we rename this to INearInteractionTouchableDirected? It's not going to be used by the volume near interaction touchable and doesn't make sense to move it into base class. I think renaming would make it easier to understand what it's for.